setup.py

```diff
@@ -1356,6 +1403,22 @@ def main():
         "utils/model_dump/*.mjs",
     ]
 
+    if BUILD_PYTORCH_USING_LIBTORCH_WHL:
```

Comment:
If this option ``BUILD_PYTORCH_USING_LIBTORCH_WHL`` is not supported yet, I think its better to add it when its ready.

Reply:
For this PR I think it's reasonable given I'm not planning on merging this one without the next one on the stack. Also I'd rather not rewrite the dependency logic.

Reply:
Though for next time you're right